src/pages/our-pets.js

```diff
@@ -1,9 +1,12 @@
 import pets from '../modules/pets-data';
 import renderPets from '../modules/render-pets';
-
+import sortPetsByName from '../modules/sort-pets';
 
 window.addEventListener('load', () => {
   const petsContent = document.querySelector('.ourpets-content');
+  const sortForm = document.querySelector('.petsForm');
+  if (!sortForm) return;
```

Comment:
Probably not the best way to put such condition before renderPets, if for some reason the sorting will be removed, code will stop on this line and no content will be rendered